src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -659,6 +659,7 @@ namespace
             QComboBox* combobox = create_combobox("engine");
             combobox->setToolTip(m_params_metadata.get_path("lighting_engine.help"));
             combobox->addItem("Unidirectional Path Tracer", "pt");
+            combobox->addItem("Bidirectional Path Tracing", "bdpt");
```

Comment:
I think this should be named `"Bidirectional Path Tracer"` for consistency.

Reply:
Good point.